### DISCUSSION TITLE
Динамическое изменение размеров карты

### DISCUSSION THREAD

**Discussion by ri4ans** [yandex-maps-unofficial/vue-yandex-maps#65] [Q&A (1.x)] [answered]
Как мне динамически изменить размер карты? В обычной yandex map у созданного экземпляра можно вызвать метод `myMap.container.fitToViewport()`. А здесь как? Подскажите пожалуйста

**Comment by daniluk4000** [comment, marked as answer]
Точно так же: вы получаете карту по событию @created, после чего можете вызвать эту функцию.

Например, так:

```vue
<template>
  <yandex-map @created="map = $event"/>
</template>

<script>
export default defineComponent({
     data() {
          map,
      },
     methods: {
          fitMap() {
            this.map?.container.fitToViewport();
          },
     }
}); 
</script>
```
